### PR TITLE
Translocation

### PR DESCRIPTION
**Pull request by JetteReeg**
minSteps and maxSteps were not references as stage-sex matrix and parameter output was not sex and stage dependent:

- adapted the output for the parameter settings and added (if applicable) sex and stage dependent minSteps and maxSteps output
- initialized both parameters as matrix (similar to maxStepsYr)
- in Species::setSteps() function: assigned the input maxSteps and minSteps according to stage and sex (0 if not applicable)
- extracted the values in Species::getSteps() according to sex and stage (0 if not applicable)

@TheoPannetier: I am unsure about the input from the batch side. In R the input is a vector of the size NSexes*NStages.

(the other commits just refer to changes in links within Readme and Contributing files.)

### CODE REVIEW

File: Model.cpp [hunk 1]
```diff
@@ -769,7 +769,7 @@ int RunModel(Landscape* pLandscape, int seqsim)
 
 }
 
-#if RS_EMBARCADERO || LINUX_CLUSTER || RS_RCPP 
+#if RS_EMBARCADERO || LINUX_CLUSTER || RS_RCPP
```
Comment (TheoPannetier) [inline Model.cpp]:
```suggestion
#if LINUX_CLUSTER || RS_RCPP
```

File: Model.cpp [hunk 2]
```diff
@@ -1687,23 +1687,46 @@ void OutParameters(Landscape* pLandscape)
...
-		outPar << "MIN. No. OF STEPS:\t " << ssteps.minSteps << endl;
-		outPar << "MAX. No. OF STEPS:\t ";
-		if (ssteps.maxSteps == 99999999) outPar << "not applied" << endl;
-		else outPar << ssteps.maxSteps << endl;
+		// ssteps = pSpecies->getSteps(0, 0);
+		//
+		// outPar << "MIN. No. OF STEPS:\t " << ssteps.minSteps << endl;
+		// outPar << "MAX. No. OF STEPS:\t ";
+		// if (ssteps.maxSteps == 99999999) outPar << "not applied" << endl;
+		// else outPar << ssteps.maxSteps << endl;
```
Comment (TheoPannetier) [inline Model.cpp]:
```suggestion
```
I f we're not using it, might as well delete it :)

File: Species.cpp [hunk 3]
```diff
@@ -116,11 +116,11 @@ Species::Species(void)
 
 	// initialise settlement parameters
 	stgDepSett = false; sexDepSett = false; indVarSett = false;
-	minSteps = 0; maxSteps = 99999999;
+
 	for (int i = 0; i < NSTAGES; i++) {
 		for (int j = 0; j < NSEXES; j++) {
 			densDepSett[i][j] = false; wait[i][j] = false; go2nbrLocn[i][j] = false; findMate[i][j] = false;
-			maxStepsYr[i][j] = 99999999;
+			maxStepsYr[i][j] = 99999999; 	minSteps[i][j] = 0; maxSteps[i][j] = 99999999;
```
Comment (TheoPannetier) [inline Species.cpp:123]:
```suggestion
			maxStepsYr[i][j] = 99999999; 	
			minSteps[i][j] = 0; 
			maxSteps[i][j] = 99999999;
```

File: Model.cpp (hunk 1 above)
Comment (TheoPannetier) [reply Model.cpp]: We're not using Embarcadero anymore, might as well drop this!

**Review by TheoPannetier — APPROVED** [review]
Looks great! I only have a couple of minor readability suggestions, please have a look before merging 👍

**Comment by TheoPannetier** [comment]
Oddly enough, the batch interface seems to allow this already (see https://github.com/RangeShifter/RangeShifter_batch/blob/29f1935905f3b56cdc39ace54bc34182a9228d4e/src/BatchMode.cpp#L5076 onwards), but the possibility of sex-dependent/stage-dependent step requirements are not mentioned anywhere in the doc. 
So integrating in the next release should only take a doc update!

File: Model.cpp (hunk 1 above)
Comment (JetteReeg) [reply Model.cpp]: agreed

File: Model.cpp (hunk 2 above)
Comment (JetteReeg) [reply Model.cpp]: true

**Pull request merged**